Sources/Extensions/SpriteKit/SKNodeExtensions.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  SpriteKitExtensions.swift
+//  SwifterSwift
+//
+//  Created by Olivia Brown on 5/28/18.
+//  Copyright © 2018 SwifterSwift
+//
+
+import SpriteKit
```

Comment:
Please wrap this by
```swift
#if canImport(SpriteKit)
...
#endif
```

Reply:
SpriteKit is also available for macOS, tvOS, and watchOS, could you please update `SpriteKitExtensions.swift` target membership to be included in other targets as well :)

Reply:
I have to learn how to figure this out from the Xcodeproj diff without actually cloning the project

Reply:
@omaralbeik I thought that I added it to all targets with https://github.com/SwifterSwift/SwifterSwift/pull/490/commits/0487ecbec8e668223d21e651f936bb4d36be1936. What am I missing?

Reply:
yeah, I'm curious too @omaralbeik. CI shows this extension being tested on 3 platforms (we don't run tests for watchOS) 🤔

Reply:
hmm, yeah I see you've added it to all targets, maybe I forgot to fetch origin, sorry 😅